Member-Travel-Fund.md

```diff
@@ -1,41 +1,61 @@
 ## Purpose
 
 To establish and administer a fund for members of the Node.js
-Foundation to travel and spread knowledge about and support the foundation
-and the Node.js Foundation projects.
+Foundation to travel, spread knowledge about, and support the Node.js 
+Foundation projects.
 
 ## Restrictions
 
 * Candidates must be Individual Members of the Node.js Foundation.
 
+### Travel Fund Administrators
+
+Members of both the TSC and CommComm are responsible for approving fund requests
+and communicating approval/denial through the PR filed by the member requesting. 
+
+Travel fund admins will consider each proposal tagged for their respective group 
+in its next regular meeting. Requests tagged for both committees must be coordinated 
+and communicated together by the Chair of each. See considerations below to set 
+yourself up for success to be approved.
+
+Linux Foundation Accounts Payable and the Node.js community manager are responsible for 
+verifying the requester is an Individual Member of the Node.js Foundation and dispersing funds,
+as required to receive funds(see Reimbursement below).
+
 ## Process
 
 Any member can apply for travel funds. Each request may take up to a month
-for the TSC to approve or decline.
+for the travel fund administrators to approve or decline.
 
+### Request
 * Open a pull request against this repository which adds an entry to the table at the bottom of this file.
  * Include the name of the event you plan to attend.
  * Include the location of the event and where you will be traveling from.
  * Include the presentation you intend to give, if applicable.
  * Include the size of the stipend you wish to receive.
   * Reimbursement stipends are expected to vary with travel distance.
+  * In the PR description, tag the group relevant to the request: @nodejs/tsc or @nodejs/commcomm-members. If unsure, include both.
 * Once the final amount spent is known, update the table again with that information.
 
-The TSC will consider each proposal in its next regular meeting. While it is
-ultimately at the TSC's discretion who to approve the following considerations
-are made.
+### Reimbursement
+Once the request has been approved, provide receipts as attachments in an email stating your name, the participation covered, and the total approved for reimbursement. Send to ap@linuxfoundation.org and cc tracyhinds@linuxfoundation.org with subject ```Node.js Member Travel Fund```. Due to privacy, the Individual Members list is not public. This team will verify the requester is on the Individual Members list before funds are dispersed. This dispersement is generally processed within 30 days. 
+
+**The following considerations are made for approval of the request:**
```

Comment:
iirc this was in the original travel fund doc (or at least the one that was pr'd to the commcomm repo), so yeah, it would make sense to include that here